CMP1903_A1_2324/Die.cs

```diff
@@ -3,21 +3,34 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Diagnostics;
 
 namespace CMP1903_A1_2324
 {
     internal class Die
-    {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
 
+    {
+        private int currentValue;
+        private static Random random;
```

Comment:
I would change these to include an underscore before them to identify that they are private variables, _currentValue and _random - naming conventions.